cli/server/server_test.go

```diff
@@ -49,15 +51,38 @@ func TestGetConfigFromContext(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, netmode.TestNet, cfg.ProtocolConfiguration.Magic)
 	})
-	t.Run("relative-path", func(t *testing.T) {
+	t.Run("relative-path-windows", func(t *testing.T) {
```

Comment:
This commit is taken from other PR. Put `Depends on #...` line in the PR message.